### ISSUE TITLE
Add Galois to txt2json.py

### ISSUE THREAD

**Issue by jowens** [gunrock/io#4]
txt2json.py does not currently support the nice Galois results I just uploaded.

**Comment by ffarhour** [comment]
I had a look at the script and I may be able to add Galois to txtjson.py.  But I will need the original txt files from the other engines as a point of comparison e.g. txt outputs from gunrock. Would I need to build gunrock and generate them myself or can I find them somewhere?

**Comment by jowens** [comment]
Not sure. @huanzhang12? @yzh?

**Comment by huanzhang12** [comment]
Hi Farhour,

In txt2json.py, starting from line 253, you will see some variable defining the .txt input file path for each engine. For example, for Ligra the path is "/data/Compare/ligra_may22-2014/ppopp16/*-result"

Hope this helps.

Thanks,
Huan

**Comment by ffarhour** [comment]
Hi @huanzhang12. The folder you mentioned is not in the repo. I tried checking mario, but couldn't find them. Are the files on mario? If so, could you direct me to where they are? thanks

**Comment by huanzhang12** [comment]
Yes, they were on mario. @yzhwang might move some of them after I wrote the script.
You can try results files for BC. I just checked and there are still there:
/data/Compare/gpu_BC/ppopp16/*.txt

**Comment by neoblizz** [comment]
Addressed in https://github.com/gunrock/io/commit/de830b302f56a7923d0fc5fbd6c0f87e36b649e5.
